packages/slice-machine/src/modules/environment/index.ts

```diff
@@ -55,10 +79,15 @@ export const selectIsSimulatorAvailableForFramework = (
   return simulatorIsSupported(store.environment.framework);
 };
 
-export const getChangelog = (
-  store: SliceMachineStoreType
-): PackageChangelog => {
-  return store.environment.changelog;
+export const getChangelog = (store: SliceMachineStoreType) => {
+  return (
+    store.environment?.changelog ?? {
```

Comment:
more conditionals have been added?

Reply:
This was because the changelog will now be undefined on first load, so we just return a default _empty_ one to not throw errors. 

I think the optional chaining on environment might not be needed though 😕